### DISCUSSION TITLE
Packer change VM Size from default Standard_D4s_v4

### DISCUSSION THREAD

**Discussion by SPALLADI** [actions/virtual-environments#5880] [Image configuration]
Hi,

I would like change the VM_size from Standard_D4s_v4 to Standard_DS1_v2. How can I achieve this?
+            "managed_image_name": "{{user `image_name`}}-{{user `image_version`}}",
+            "managed_image_resource_group_name": "{{user `resource_group`}}",
             "virtual_network_name":  "{{user `virtual_network_name`}}",
             "virtual_network_resource_group_name": "{{user `virtual_network_resource_group_name`}}",
             "virtual_network_subnet_name": "{{user `virtual_network_subnet_name`}}",
             "os_type": "Linux",
             "image_publisher": "canonical",
             "image_offer": "0001-com-ubuntu-server-focal",
-            "image_sku": "18.04-LTS",
+            "image_sku": "18.04-LTS-gen2",
             "os_disk_size_gb": "86"
             "managed_image_name" : "self-hosted-ubuntu1804"
             "vm_size": "Standard_DS1_v2",
         }

**Comment by miketimofeev** [comment]
@SPALLADI `"vm_size": "Standard_DS1_v2"`, should work

**Reply by SPALLADI** [reply]
Thank you, as you can see I have provided as part of build command & in the patch json file
